lib/note-list/decorators.tsx

```diff
@@ -14,6 +14,9 @@ export const decorateWith = (decorators, text) =>
               ? new RegExp(escapeRegExp(searchText), 'gi')
               : filter;
 
+          if (output?.length === 0) {
+            return [];
+          }
```

Comment:
it seems like the only way to get an empty string here for `output` is if the input `text` was also an empty string. maybe it makes more sense to abort early and skip all this if we know the input is empty since we can't decorate "nothing"

```ts
export const decorateWith = (decorators, text) =>
	decorators.length > 0 && text.length > 0
		? decorators.reduce(…).map(…)
		: text;
```

Reply:
Thanks, updated